analytic_engine/Cargo.toml

```diff
@@ -25,7 +25,9 @@ workspace = true
 workspace = true
 
 [features]
+default = ["wal-table-kv"]
```

Comment:
I think we should use rocksdb wal as default wal to ensure the first time trial to be friendly for users.

Reply:
The origin we go here is to opt-out rocksdb wal impl. But it's OK to work as explicit instead of by default.

I'm open to other advice for what is desired here @jiacai2050.

BTW, table-kv can use memory impl as default for smoothy first time trial?

Reply:
> BTW, table-kv can use memory impl as default for smoothy first time trial?
If so, it may be confusing about that the data is lost after restarting the server.

Reply:
@Rachelint @jiacai2050 Any suggestions about the default wal implemenation?

Reply:
I prefer default to rocksdb implementation, as it was before. 

Memory impl only for testing and may cause confusion if users restart server and found data is lost.

Reply:
@tisonkun After some discussions, we guess this is a proper way to set the feature gates:
- The default feature should include all the WAL implementations, to avoid any possible confusion about data loss, config not working, etc.
- A specific WAL implementation can be chosen for a fast deployment. (Currently, in our prod env, there is no need to build ceresdb with rocksdb).

And in order to build ceresdb without rocksdb by default, I guess another simple WAL implementation based on file system should be provided.

Reply:
OK. Let me try to modifiy this patch with the resolution above :D